tests/test_varnameinfo.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from pyaerocom.exceptions import VariableDefinitionError
+from pyaerocom.varnameinfo import VarNameInfo
+
+
+@pytest.mark.parametrize(
+    "var_name, contains_numbers, contains_wavelength_nm, is_wavelength_dependent",
+    [
+        ("concpm10", True, True, False),
+        ("wetoxn", False, False, False),
+        ("od550gt1aer", True, True, True),
+    ],
+)
+def test_varnameinfo(var_name, contains_numbers, contains_wavelength_nm, is_wavelength_dependent):
+    var_name_info = VarNameInfo(var_name)
+    assert isinstance(var_name_info, VarNameInfo)
+    assert var_name_info.contains_numbers == contains_numbers
+    assert var_name_info.contains_wavelength_nm == contains_wavelength_nm
+    assert var_name_info.is_wavelength_dependent == is_wavelength_dependent
+    assert isinstance(var_name_info.__str__(), str)
```

Comment:
`__str__()` is called when converting a class into a `str`, so maybe this would be better 
``` python
    assert isinstance(str(var_name_info), str)
```
or 
``` python
    assert hasattr(var_name_info, "__str__")
```
but I'm not sure....

Reply:
I want to actually execute the `__str__()` function though. I don't think these alternatives will do that

Reply:
`str(object)` will call `object.__str__()`, but it is up to you

Reply:
Okay, first option it is then :)